Fix Copilot "Trigger inline suggestion" shortcut displaying backslash incorrectly

ThijsBoehme opened the pull request:
The backslash in the shortcut listing is displaying as `Alt+</kbd>` rather than `Alt+\`

### Why:

Closes: #28543

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Escaping the backslash in the shortcut should fix the display.

Before:
![Screenshot 2023-09-27 at 14 53 54](https://github.com/github/docs/assets/12415682/0073d7a4-bb5b-4240-a30d-4feb3c1e31a8)

After:
![Screenshot 2023-09-27 at 14 53 42](https://github.com/github/docs/assets/12415682/d3da6f83-4bdf-422b-8a15-05358fee9693)

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

ThijsBoehme committed:
Fix backslash shortcut displaying incorrectly

The backslash in the shortcut listing is displaying as `Alt+</kbd>` rather than `Alt+\`

felicitymay reviewed (dismissed):
Many thanks for spotting this display error and fixing it ✨ 

We should be able to merge your pull request shortly.

felicitymay committed:
Merge branch 'main' into patch-1

felicitymay committed:
Escape a couple more instances

The table is shown for some of the other IDEs too. Adding escaping for the back-slashes in those situations too.

felicitymay committed:
Merge branch 'main' into patch-1

felicitymay committed:
Merge branch 'main' into patch-1
